Lib/test/test_io.py

```diff
@@ -3508,7 +3508,13 @@ def _check(dec):
         _check(dec)
 
 class CIncrementalNewlineDecoderTest(IncrementalNewlineDecoderTest):
-    pass
+    def test_uninitialized(self):
```

Comment:
Add cpython_only? This is an implementation detail, if implement the constructor as `__new__` rather than `__init__`, the following errors will be not raised.